base/applications/dxdiag/system.c

```diff
@@ -153,6 +153,29 @@ VOID GetSystemCPU(WCHAR *szBuffer)
     }
 }
 
+static
+unsigned
```

Comment:
SIZE_T
or DWORD ... but then I think you'll have tkreuzer "fix for 64-bit" the code ^^

Reply:
Yeah I already got the idea, I've seen @tkreuzer's fixes :sweat_smile: